polkadot-test-service/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+[package]
+name = "polkadot-test-service"
```

Comment:
it seems somewhat redundant to put this in a directory starting with `polkadot`. What do you think about just calling the directory `test-service`? Also, maybe put it in `node/` instead of at the root. That will keep things more organized

Reply:
It definitely is... when I first named it I called it just "test" but it was super confusing with the tests directory.

I'm not familiar with the "node" directory. I think it's newish? I see it's been introduced in early June. What should we put there?

(Meanwhile I will follow your suggestion and move it already)

Reply:
Yep, the `node` crate is new and is meant to hold all things related to the node. At the end I think I'd like to have only `node`/`core`/`primitives` at the root.

Reply:
I see that we have a `test-helpers` directory. Wouldn't it be even better if I moved it to there in "service" (crate name would be: polkadot-service-test-helpers)